src/main/java/org/intellij/sdk/codesync/files/UserFile.java

```diff
@@ -1,22 +1,18 @@
 package org.intellij.sdk.codesync.files;
 
+import org.intellij.sdk.codesync.DataClass.UserTable;
 import org.intellij.sdk.codesync.exceptions.FileNotCreatedError;
 import org.intellij.sdk.codesync.exceptions.InvalidYmlFileError;
 import org.intellij.sdk.codesync.utils.CommonUtils;
 
 import java.io.*;
-import java.util.HashMap;
 import java.util.Map;
 
-import static org.intellij.sdk.codesync.Constants.USER_FILE_PATH;
+public class UserFile{
```

Comment:
I think we can complelty remove this class now. We can use `public class User` instead